src/shared/components/comment/comment-node.tsx

```diff
@@ -112,6 +112,13 @@ interface CommentNodeProps {
   onPurgeComment(form: PurgeComment): Promise<void>;
 }
 
+function handleToggleViewSource(i: CommentNode) {
+  i.setState(({ viewSource, ...restPrev }) => ({
+    viewSource: !viewSource,
+    ...restPrev,
```

Comment:
Side point but is it really necessary to set the rest of the state always?

Reply:
I think.  Hard to tell due to inferno's lack of docs.